specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-02-02/blob.json

```diff
@@ -5654,6 +5657,16 @@
                   "name": "SyncCopyStatusType",
                   "modelAsString": false
                 }
+              },
+              "Content-MD5": {
+                "type": "string",
+                "format": "byte",
+                "description": "This response header is returned so that the client can check for the integrity of the copied content. This header is only returned if the source content MD5 was specified."
+              },
+              "x-ms-content-crc64": {
+                "type": "string",
+                "format": "byte",
+                "description": "This response header is returned so that the client can check for the integrity of the copied content."
```

Comment:
Do we have any APIs with both of these yet?  This might need a bit of discussion to get right for .NET.

Reply:
I think we should add "x-ms-client-name": "Content-CRC64" to x-ms-content-crc64.

Reply:
I was holding off on the nice name for consistency reasons, as the other instances didn't have it.

Reply:
@tg-msft stage_block_from_url has these two headers https://github.com/zezha-msft/azure-rest-api-specs/blob/5504d99b737b1deaa7b6ce4518e90689e804c975/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-02-02/blob.json#L6167-L6176